README.md

```diff
@@ -20,9 +20,9 @@ From **provisioning** and **configuration** to **orchestration** and **monitorin
     * [foreman-users](https://groups.google.com/forum/?fromgroups#!forum/foreman-users)
     * [foreman-dev](https://groups.google.com/forum/?fromgroups#!forum/foreman-dev)
 
-Using Puppet, [Chef](http://github.com/theforeman/foreman_chef), [Salt](http://github.com/theforeman/foreman_salt) and Foreman's [smart proxy](http://github.com/theforeman/smart-proxy) architecture, you can easily automate repetitive tasks, quickly deploy applications, and proactively manage change, both on-premise with VMs and bare-metal or in the cloud.
+Using Puppet, [Ansible](https://theforeman.org/plugins/foreman_ansible/), [Chef](https://theforeman.org/plugins/foreman_chef/), [Salt](https://theforeman.org/plugins/foreman_salt/) and Foreman's [smart proxy](http://github.com/theforeman/smart-proxy) architecture, you can easily automate repetitive tasks, quickly deploy applications, and proactively manage change, both on-premise with VMs and bare-metal or in the cloud.
```

Comment:
We should either link to all of the Github pages of Puppet, Ansible, Salt and Smart-Proxy or all links should point to the manual. I'd prefer the former, but don't have a strong preference.